ledfx/effects/random.py

```diff
@@ -0,0 +1,91 @@
+import random
+import timeit
+
+import numpy as np
+import voluptuous as vol
+
+from ledfx.color import parse_color, validate_color
+from ledfx.effects.temporal import TemporalEffect
+
+
+class RandomEffect(TemporalEffect):
+    """
+    Randomly activates sections
+    """
+
+    NAME = "Random"
+    CATEGORY = "Non-Reactive"
+
```

Comment:
add a hidden key for "flip" as it won't do anything for your effect, it has no directionality, so just hide it.